.github/workflows/run-td-build.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches:
       - main
       - release/**
+      - prerelease-beta/**

```

Comment:
Same here, this looks unrelated to PR description:
![image](https://user-images.githubusercontent.com/2684369/161788840-39dd9d5a-3df4-4da0-aafc-860cab8e35fa.png)